.github/workflows/release.yml

```diff
@@ -50,3 +54,59 @@ jobs:
           files: |
             pie.phar
             pie.phar.asc
+
+  docker-binary-only-image:
+    needs: release-phar
+    name: Docker binary-only image
+    runs-on: ubuntu-latest
+    if: ${{ startsWith(github.ref, 'refs/tags/') }}
+
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Log in to the Container registry
+        uses: docker/login-action@v3
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Make image lowercase
+        run: |
+          echo IMAGE_NAME_LOWER=$(echo ${{ env.IMAGE_NAME }} | tr '[:upper:]' '[:lower:]') >> ${GITHUB_ENV}
+
+      - name: Extract metadata (tags, labels) for Docker
+        id: meta
+        uses: docker/metadata-action@v5
+        with:
+          flavor: |
+            latest=true
+          images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME_LOWER }}
+          tags: |
+            type=semver,pattern={{version}}
+            type=semver,pattern={{major}}.{{minor}}
+            type=semver,pattern={{major}}
+
+      - name: Build and push Docker image
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          platforms: linux/amd64,linux/arm64
```

Comment:
Is there a way we can build this architecture-agnostic (rather than explicitly building for arm64/amd64/etc) :thinking:

Reply:
As pointed out [here](https://github.com/php/pie/pull/122#discussion_r1853101732), we can't do it (or at least I don't know how). Is it a problem, though?

Reply:
I was hoping you could build for `noarch` architecture or something to indicate that a single manifest can operate on any platform, since the only content of the image (the PHAR) is entirely platform agnostic. Not a major problem, but it creates overhead for something that should otherwise be a single step >.<